app/src/components/ColourKey.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./Component.css";
+
+const ColourKey: React.FC = () => {
+    const statuses: string[] = ["Nominal", "Moderate", "Critical"];
+    let colour: string;
+    return (
+        <div className="flex flex-row justify-evenly m-auto">
+            {statuses?.map(function (status) {
+                if (status == "Nominal") {
+                    colour = "green";
+                } else if (status == "Moderate") {
+                    colour = "yellow";
+                } else if (status == "Critical") {
+                    colour = "red";
+                }
```

Comment:
For these, I noticed that we have a lot of 'Nominal', 'Moderate' and 'Critical' if else statements. 
Might be good to create a helper, so that we can simply input a status, and get out a color. Since there are multiple files which check for these, it would be good to create a new class entirely which can be called